test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -418,7 +418,7 @@ Describe "TabCompletion" -Tags CI {
                 @{ inputStr = 'gmo -list PackageM'; expected = 'PackageManagement'; setup = $null }
                 @{ inputStr = 'gcm -Module PackageManagement Find-Pac'; expected = 'Find-Package'; setup = $null }
                 @{ inputStr = 'ipmo PackageM'; expected = 'PackageManagement'; setup = $null }
-                @{ inputStr = 'Get-Process powersh'; expected = 'powershell'; setup = $null }
+                @{ inputStr = 'Get-Process pws'; expected = 'pwsh'; setup = $null }
```

Comment:
A typo -- `pws`

Reply:
This is for the tab completion test so `pws` completes to `pwsh` since there isn't necessarily a `powershell` process anymore and `pw` might resolve to something else

Reply:
Ah, yes, my mistake.